lib/sorcery/controller/submodules/session_timeout.rb

```diff
@@ -36,7 +36,7 @@ def invalidate_active_sessions!
             return unless current_user.present?
 
             current_user.send(:invalidate_sessions_before=, Time.now.in_time_zone)
-            current_user.save
+            current_user.save(validate: false)
```

Comment:
Do we need to skip validation here?

Reply:
If we don't, password is required. Additionally it should be independent from other validations I think

Reply:
I'm pretty sure I noticed this myself while working on the v1 rework. It doesn't make sense for us to care about any validations, as we're editing a value outside of those normal loops. I'll have to sanity check that before merging that in though.

Also, @wojciech-rzepecki, normally you include a proc that checks for if the password is being set or changed so that it doesn't interfere during normal editing. (e.g., a user editing their timezone)

For example:
```ruby
validates :password,
  presence: true,
  confirmation: true,
  if: -> { new_record? || changes[:crypted_password] }

validates :password_confirmation,
  presence: true,
  if: -> { new_record? || changes[:crypted_password] }
```